src/apis/categories/index.mock.ts

```diff
@@ -1,9 +1,10 @@
 import { rest } from 'msw';
+import { CATEGORIES_PATHS } from '@apis/path';
 
-import { getCategoriesPath } from './useGetCategorys';
+const BASE_URL = process.env.REACT_APP_BASE_URL;
 
 export const categoriesMockHandler = [
-  rest.get(getCategoriesPath(), (_, res, ctx) => res(ctx.json(CATEGORIES_RESPONSE_DATA))),
+  rest.get(`${BASE_URL}${CATEGORIES_PATHS.CATEGORIES}`, (_, res, ctx) => res(ctx.json(CATEGORIES_RESPONSE_DATA))),
```

Comment:
```suggestion
  rest.get(`*${CATEGORIES_PATHS.CATEGORIES}`, (_, res, ctx) => res(ctx.json(CATEGORIES_RESPONSE_DATA))),
```

BASE_URL 명시하는 방법도 있는데 `*` wildcard 사용해서 모든 도메인에 대해 허용하는 방법도 있다는 점 참고해주시면 될 거 같아요.